cmd/crc/cmd/config/unset.go

```diff
@@ -21,6 +22,9 @@ func configUnsetCmd(config config.Storage) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			telemetry.SetContextProperty(cmd.Context(), "key", args[0])
```

Comment:
Don't we want to also know which key was attempted to be (un)set in failure cases?

Reply:
I was tempted but what if the key is the value and we collect bad data? `crc config set mypassword`. At least the key is always validated here.

Reply:
ah good point